packages/backend/package.json

```diff
@@ -52,12 +52,12 @@
     }
   },
   "scripts": {
-    "build": "vite build",
+    "build": "yarn openapi-typescript ../../api/openapi.yaml -o src-generated/openapi.ts && vite build",
```

Comment:
maybe use a "generate-open-ai" script and we call it from build and watch ?

Reply:
Fixed by following commit